python/test/t_PythonFunction_multiprocessing.expout

```diff
@@ -0,0 +1,2 @@
+T sequential =  62 (s)
```

Comment:
dont expect the time to be the same across different build hosts!

Reply:
Hello Julien, You are right there was a regression due to forgotten a case. I pushed a correction and i am preparing a benchmark with more explanation for the next pull request. Thanks